### PR TITLE
fix: add disk speed annotation [2]

### PR DESCRIPTION
**Pull request by minz1027**
- add disk speed annotation / node selector
- if annotation not specified, can go to any node. If annotation specified, will go to the desired node with proper resources 

Blocked by: https://github.com/screwdriver-cd/executor-base/pull/51
Related to: https://github.com/screwdriver-cd/screwdriver/issues/1463

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -603,6 +635,40 @@ describe('index', () => {
             });
         });
 
+        it('sets disk speed toleration and node affinity when diskspeed is HIGH', () => {
+            fakeStartConfig.annotations = { 'screwdriver.cd/diskspeed': 'high' };
```
Comment (tkyi) [inline test/index.test.js]:
```suggestion
            fakeStartConfig.annotations = { 'screwdriver.cd/diskSpeed': 'high' };
```

Comment (minz1027) [reply test/index.test.js]: addressed 👍

File: index.js [hunk 2]
```diff
@@ -276,18 +279,29 @@ class K8sVMExecutor extends Executor {
...
 
-            hoek.merge(nodeSelectors, this.nodeSelectors);
+            hoek.merge(nodeSelectors, diskSelectors);
+        }
 
-            setNodeSelector(podConfig, nodeSelectors);
-        } else {
-            setNodeSelector(podConfig, this.nodeSelectors);
+        if (this.diskSpeedLabel) {
+            const speedValues = ['turbo', 'high'];
```
Comment (jithin1987) [inline index.js]: move defined constants out of the function.

Comment (minz1027) [reply index.js]: addressed

**Pull request merged**